Scenarios/ALKS_Scenario_4.2_4_MultipleBlockingTargets_TEMPLATE.xosc

```diff
@@ -76,7 +102,7 @@
           <PrivateAction>
             <TeleportAction>
               <Position>
-                <LanePosition roadId="0" laneId="$Ego_InitPosition_LaneId" offset="0.0" s="515.0"></LanePosition>
+                <LanePosition roadId="0" laneId="$Ego_InitPosition_LaneId" offset="0.0" s="${$TargetBlocking_InitPosition_LongitudinalOffset_m + 15.0}"></LanePosition>

```

Comment:
Works for all combinations except two consecutive trucks. These would overlap. But since this isn't a menaingful test-case (first target should be smaller than second target), this is fine.